movingpandas/tests/test_trajectory.py

```diff
@@ -181,25 +181,30 @@ def test_add_direction_overwrite_raises_error(self):
     def test_add_speed(self):
         traj = make_traj([Node(0, 0), Node(6, 0, second=1)])
         traj.add_speed()
-        assert traj.df[SPEED_COL_NAME].tolist() == [6.0, 6.0]
+        assert traj.df["speed"].tolist() == [6.0, 6.0]
 
     def test_add_speed_without_crs(self):
         traj = make_traj([Node(0, 0), Node(6, 0, second=1)], crs=None)
         traj.add_speed()
-        assert traj.df[SPEED_COL_NAME].tolist() == [6.0, 6.0]
+        assert traj.df["speed"].tolist() == [6.0, 6.0]
 
     def test_add_speed_can_overwrite(self):
         traj = make_traj([Node(0, 0), Node(6, 0, second=1)])
         traj.add_speed()
         traj.add_speed(overwrite=True)
-        assert traj.df[SPEED_COL_NAME].tolist() == [6.0, 6.0]
+        assert traj.df["speed"].tolist() == [6.0, 6.0]
 
     def test_add_speed_overwrite_raises_error(self):
         traj = make_traj([Node(0, 0), Node(6, 0, second=1)])
         traj.add_speed()
         with pytest.raises(RuntimeError):
             traj.add_speed()
 
+    def test_add_speed_with_name(self):
+        traj = make_traj([Node(0, 0), Node(6, 0, second=1)])
+        traj.add_speed(name="speed2")
+        assert "speed2" in traj.df.columns
```

Comment:
We should also check that the values in a potentially already existing "speed" column are not changed and that the values in "speed2" are correct.